backend-checks/src/test/scala/uk/gov/nationalarchives/tdr/localaws/backendchecks/checks/metadata/FakeChecksumTest.scala

```diff
@@ -0,0 +1,31 @@
+package uk.gov.nationalarchives.tdr.localaws.backendchecks.checks.metadata
+
+import java.nio.file.Paths
+
+import org.scalatest.matchers.should
+import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatest.propspec.AnyPropSpec
+
+class FakeChecksumTest extends AnyPropSpec with TableDrivenPropertyChecks with should.Matchers {
+
+  case class ChecksumExample(fileName: String, expectedChecksum: String)
+  val examples = Table(
+    "Antivirus metadata examples",
```

Comment:
Should read: "**Checksum** metadata examples"

Reply:
Fixed, thanks. I must have been really careless yesterday. 😬